scripts/axiom-diagnostics.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/sh
 
+
+if test -n "$4"
+then
+   GROUPBYFLAG=$4
```

Comment:
Please use only 2 spaces on indentation to stay consistent with the rest of the file. Also, why is the variable name different here than in the makefile?

```suggestion
  group_by_mod=$4
```